src/OSPSuite.R/Domain/SimulationBatch.cs

```diff
@@ -148,12 +149,15 @@ public SimulationResults Run(SimulationBatchRunValues simulationBatchRunValues)
       {
          _simModelBatch.UpdateParameterValues(simulationBatchRunValues.Values);
          _simModelBatch.UpdateInitialValues(simulationBatchRunValues.MoleculeValues);
-         var simulationResults = _simModelBatch.RunSimulation();
+         var simulationRunResults = _simModelBatch.RunSimulation();
+
+         if (!simulationRunResults.Success)
+            throw new OSPSuiteException(simulationRunResults.Error);
```

Comment:
Unsuccessful run must be signaled by exception

Reply:
yep